src/neo-vm/Types/StackItem.cs

```diff
@@ -59,6 +59,10 @@ public virtual StackItem ConvertTo(StackItemType type)
             throw new InvalidCastException();
         }
 
+        internal virtual void Cleanup()
```

Comment:
If StackItem implements `IDisposable`, it could be used with using blocks

Reply:
We can't use a `using` statement with a `StackItem`. Because they are always created and pushed onto the stack, then cleaned up in the `ReferenceCounter`, or waiting for the GC.